pkg/ddevapp/templates.go

```diff
@@ -33,10 +32,11 @@ services:
     ports:
       - "80"
       - "{{ .mailhogport }}"
-    working_dir: "/var/www/html/docroot"
+    working_dir: "/var/www/html"
```

Comment:
I suspect this should be the docroot; It just used to be nice to be able to `ddev ssh` and immediately do a drush command. Also... `ddev exec drush ...` doesn't work now.